src/main/java/io/jenkins/plugins/artifact_manager_jclouds/BlobStoreProvider.java

```diff
@@ -69,6 +70,10 @@
     @NonNull
     public abstract BlobStoreContext getContext() throws IOException;
 
+    /** Return the jclouds Blobstore for working with blob. */
+    @NonNull
+    public abstract BlobStore getBlobStore() throws IOException;
```

Comment:
Maybe better to make this return `getContext().getBlobStore()` by default.

Reply:
agreed